modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/file/FilePageStoreManager.java

```diff
@@ -1012,7 +1009,8 @@ else if (file.getName().startsWith(CACHE_GRP_DIR_PREFIX))
         return Arrays.stream(dir.listFiles())
             .sorted()
             .filter(File::isDirectory)
-            .filter(f -> f.getName().startsWith(CACHE_DIR_PREFIX) || f.getName().startsWith(CACHE_GRP_DIR_PREFIX))
+            .filter(f -> f.getName().startsWith(CACHE_DIR_PREFIX) || f.getName().startsWith(CACHE_GRP_DIR_PREFIX) ||
+                f.getName().startsWith(MetaStorage.META_STORAGE_DIR_NAME))
```

Comment:
may be replaced with `f.getName().equals(MetaStorage.META_STORAGE_DIR_NAME)`

Reply:
Fixed.